projects/1.20/assets/maturi-delight/maturidelight/lang/zh_cn.json

```
@@ -0,0 +1,21 @@
+{
+    "item.maturidelight.suspicious_item": "这个章鱼烧上放了什么东西……",
+    "item.maturidelight.takoyaki_raw": "生章鱼烧",
+    "item.maturidelight.takoyaki": "章鱼烧",
+    "item.maturidelight.takoyaki_soysauce": "淋有酱油的章鱼烧",
+    "item.maturidelight.okonomiyaki_raw": "生什锦烧",
+    "item.maturidelight.okonomiyaki": "什锦烧",
+    "item.maturidelight.okonomiyaki_soysauce": "淋有酱油的什锦烧",
+    "item.maturidelight.dough_raw": "生面团",
+    "block.maturidelight.takoyaki_maker": "章鱼烧器",
+    "block.maturidelight.plate": "铁板",
+    "itemGroup.maturidelight": "祭祀乐事",
+    "advancements.maturidelight.root.title": "祭祀乐事",
+    "advancements.maturidelight.root.description": "认识祭祀活动的食物！",
+    "advancements.maturidelight.making_food.title": "制作美食",
+    "advancements.maturidelight.making_food.description": "用铁板和章鱼烧器做些什么",
+    "advancements.maturidelight.extra_topping.title": "添加调料",
```

```suggestion
    "advancements.maturidelight.extra_topping.title": "添加蘸料",
```
`トッピング`即英语`topping`，可译作`蘸料`、`浇头`